README.md

```diff
@@ -1,6 +1,6 @@
 # ensembl-io
 
-[![Build Status](https://travis-ci.org/Ensembl/ensembl-io.svg?branch=master)][travis]
+[![Build Status](https://travis-ci.org/Ensembl/ensembl-io.svg?branch=release/105)][travis]
```

Comment:
Looks like this snuck in, I suspect when changing the base branch. Should be `branch=master`

Reply:
Sorry, you mean I should merge my local branch into master and then do the pull request between the two master branches?

Reply:
Yes, I think that (merging into your fork's master, then making the PR between your fork's master and this master) will fix the README issue. At the very least, it will give you an opportunity to make sure that the README is correct for master before opening the PR.